examples/plot_visualize_bad_epochs.py

```diff
@@ -14,32 +14,23 @@
 # sphinx_gallery_thumbnail_number = 2
 
 # %%
-# First, we download the data from OpenfMRI. We will download the tarfile,
-# extract the necessary files and delete the tar from the disk
+# First, we download the data from OpenfMRI which is hosted on OpenNeuro.
+# We will do this using ``openneuro-py``.
 
 import os
-import tarfile
-
+import openneuro as on
```

Comment:
I would not start with these abbreviations that nobody knows :)

just import openneuro and accept that it'll be a slightly longer line ... but then you won't have to look back to the imports every time you wonder what `on.xxx` means